src/main/java/com/sforce/cd/apexUnit/client/QueryConstructor.java

```diff
@@ -48,9 +48,10 @@
 	 * 
 	 * @return - Query to fetch apex classes as String
 	 */
-	public static String generateQueryToFetchApexClassesBasedOnRegex(String regex) {
+	public static String generateQueryToFetchApexClassesBasedOnRegex(String namespace, String regex) {
```

Comment:
@adarsh-ramakrishna-sfdc  Yes we are considering namespace as null (default) if namespace is not specified. But instead of hard-coding it, I have added it as a parameter to the function that generates query.